lib/VSCode/navigation/TabBar.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import pythonIcon from '../images/python.png';
+
+export const TabBar = ({ view }) => {
```

Comment:
Assuming I use HashRouter as I commented in the related issue, maybe "view" can be renamed to "filename". Then it can be up to the router to describe what is put in the TabBar.